source/jormungandr/jormungandr/parking_space_availability/car/divia.py

```diff
@@ -40,18 +40,26 @@
 
 class DiviaProvider(CommonCarParkProvider):
 
-    def __init__(self, url, operators, dataset, timeout=1, feed_publisher=DEFAULT_DIVIA_FEED_PUBLISHER, **kwargs):
+    def __init__(self, url, operators, dataset, timeout=1, feed_publisher=DEFAULT_DIVIA_FEED_PUBLISHER,
+                 attr_mapping=None, **kwargs):
 
         self._feed_publisher = FeedPublisher(**feed_publisher) if feed_publisher else None
         self.provider_name = 'DIVIA'
+        if attr_mapping is None:
+            attr_mapping = ['numero_parking', 'nombre_places_libres', 'nombre_places']
+
+        self.park = attr_mapping[0]
+        self.available = attr_mapping[1]
+        self.nb_places = attr_mapping[2]
```

Comment:
I don't really like the idea of mapping as configuration
I would prefer that "kind of config" to be done at "build time", something like a factory of class: 
```
DiviaPRParking = ODSParking("numero_parking", "nombre_places_libres", "nombre_places")
DiviaParkingOther = ODSParking("nb_parking", "nb_places_libres", "nb_places")
```
And use this generated class in the configuration

Reply:
@kinnou02 I did think about your solution and I hesitated a lot...  my argument to do that in conf would be that if Divia provides a new dataset with a new format(hope they won't), we don't need to do dev but only updated the deployment conf...